pkg/apis/triggers/v1alpha1/event_listener_validation_test.go

```diff
@@ -181,6 +181,14 @@ func Test_EventListenerValidate(t *testing.T) {
 										Effect:   "NoSchedule",
 									}},
 									NodeSelector: map[string]string{"beta.kubernetes.io/os": "linux"},
+									Volumes: []corev1.Volume{{
```

Comment:
could we move this to its own test case "Valid EventListener with TLS cert volumes" or something like that? Makes it easier to understand why this was added in the future.

Reply:
Yes i agree with you
Added new testcase for env related as we don't have volume support now